presto-prometheus/src/test/java/io/prestosql/plugin/prometheus/TestPrometheusQueryVectorResponseParse.java

```diff
@@ -27,6 +27,7 @@
 import static org.testng.Assert.assertNotNull;
 import static org.testng.Assert.assertTrue;
 
+@Test(singleThreaded = true) // see @BeforeMethod
 public class TestPrometheusQueryVectorResponseParse
 {
     private InputStream promVectorResponse;
```

Comment:
while you are here :)

`promVectorResponse` is not closed

Reply:
chose to ignore for now